### PR TITLE
[DependencyInjection] Document `ServiceConfigurator` remove method

### LINKED ISSUE #14854: [DependencyInjection] Add a remove() method to the PHP configurator
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#39806
| PR author(s) | @dunglas
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by alamirault**
Continue work started by @94noni in https://github.com/symfony/symfony-docs/pull/17098

Fix #14854

### CODE REVIEW

File: service_container/remove.rst
Comment (OskarStark) [inline service_container/remove.rst:1]: Ho can a user reach this page? Is there an include? I can't find one 🤔

Comment (alamirault) [reply service_container/remove.rst:1]: I thought, it's dynamically added to the menu or in Learn more ?

(service_container/calls.rst has no ref but it's in "Learn more")

However, if you see a better place, we can move it

**Comment by javiereguiluz** [comment]
Antoine, thanks for this contribution. However, a few years ago we decided to no longer use short pages in the docs (they may look convenient to document things ... but they are confusing and cumbersome for readers, which are continuously moving from one page to another).

Removing a service from the container looks like a basic feature of it, so maybe we could move this to a new section in the main `/service_container.rst` article. Thanks.

**Comment by javiereguiluz** [comment]
This is now merged! While merging, I did the change I mentioned earlier: remove the new short article and move its contents to the main DI article. Thanks.

**Comment by alamirault** [comment]
Thank you Javier !

**Pull request merged**